### PR TITLE
More localizations for Material

### PR DESCRIPTION
**Pull request by HansMuller**
This covers all the obvious remaining strings used in the material library except for a few from paginated_data_table (they'll come next).

### CODE REVIEW

File: packages/flutter/lib/src/material/i18n/material_de.arb [hunk 1]
```diff
@@ -3,5 +3,15 @@
...
+  "licensesPageTitle": "Lizenz",
+  "cancelButtonLabel": "STORNIEREN",
+  "closeButtonLabel": "SCHLIESSEN",
+  "continueButtonLabel": "FORTSETZEN",
+  "copyButtonLabel": "KOPIEREN",
+  "cutButtonLabel": "SCHNEIDEN",
+  "okButtonLabel": "OK",
+  "pasteButtonLabel": "Paste",
+  "selectAllButtonLabel": "ALLE AUSWÄHLEN",
+  "viewLicensesButtonLabel": "ANLAGEN LIZENZEN"
```
Comment (goderbauer) [inline packages/flutter/lib/src/material/i18n/material_de.arb]: LIZENZEN ANZEIGEN

File: packages/flutter/lib/src/material/i18n/material_de.arb [hunk 2]
```diff
@@ -3,5 +3,15 @@
...
+  "previousMonthTooltip": "Letzter Monat",
+  "licensesPageTitle": "Lizenz",
+  "cancelButtonLabel": "STORNIEREN",
+  "closeButtonLabel": "SCHLIESSEN",
+  "continueButtonLabel": "FORTSETZEN",
+  "copyButtonLabel": "KOPIEREN",
+  "cutButtonLabel": "SCHNEIDEN",
+  "okButtonLabel": "OK",
+  "pasteButtonLabel": "Paste",
+  "selectAllButtonLabel": "ALLE AUSWÄHLEN",
```
Comment (goderbauer) [inline packages/flutter/lib/src/material/i18n/material_de.arb]: ALLES AUSWÄHLEN

File: packages/flutter/lib/src/material/i18n/material_de.arb [hunk 3]
```diff
@@ -3,5 +3,15 @@
...
-  "previousMonthTooltip": "Letzter Monat"
+  "previousMonthTooltip": "Letzter Monat",
+  "licensesPageTitle": "Lizenz",
+  "cancelButtonLabel": "STORNIEREN",
+  "closeButtonLabel": "SCHLIESSEN",
+  "continueButtonLabel": "FORTSETZEN",
+  "copyButtonLabel": "KOPIEREN",
+  "cutButtonLabel": "SCHNEIDEN",
+  "okButtonLabel": "OK",
+  "pasteButtonLabel": "Paste",
```
Comment (goderbauer) [inline packages/flutter/lib/src/material/i18n/material_de.arb]: EINFÜGEN

File: packages/flutter/lib/src/material/i18n/material_de.arb [hunk 4]
```diff
@@ -3,5 +3,15 @@
...
   "closeButtonTooltip": "Schließen ",
   "nextMonthTooltip": "Nächster Monat",
-  "previousMonthTooltip": "Letzter Monat"
+  "previousMonthTooltip": "Letzter Monat",
+  "licensesPageTitle": "Lizenz",
+  "cancelButtonLabel": "STORNIEREN",
+  "closeButtonLabel": "SCHLIESSEN",
+  "continueButtonLabel": "FORTSETZEN",
+  "copyButtonLabel": "KOPIEREN",
+  "cutButtonLabel": "SCHNEIDEN",
```
Comment (goderbauer) [inline packages/flutter/lib/src/material/i18n/material_de.arb]: AUSSCHNEIDEN

File: packages/flutter/lib/src/material/i18n/material_de.arb [hunk 5]
```diff
@@ -3,5 +3,15 @@
   "backButtonTooltip": "Zurück",
   "closeButtonTooltip": "Schließen ",
   "nextMonthTooltip": "Nächster Monat",
-  "previousMonthTooltip": "Letzter Monat"
+  "previousMonthTooltip": "Letzter Monat",
+  "licensesPageTitle": "Lizenz",
+  "cancelButtonLabel": "STORNIEREN",
```
Comment (goderbauer) [inline packages/flutter/lib/src/material/i18n/material_de.arb]: ABBRECHEN

File: packages/flutter/lib/src/material/i18n/material_de.arb [hunk 6]
```diff
@@ -3,5 +3,15 @@
   "backButtonTooltip": "Zurück",
   "closeButtonTooltip": "Schließen ",
   "nextMonthTooltip": "Nächster Monat",
-  "previousMonthTooltip": "Letzter Monat"
+  "previousMonthTooltip": "Letzter Monat",
+  "licensesPageTitle": "Lizenz",
```
Comment (goderbauer) [inline packages/flutter/lib/src/material/i18n/material_de.arb]: Lizenzen (if this should be plural like in english)

File: packages/flutter/lib/src/material/i18n/material_de.arb [hunk 7]
```diff
@@ -3,5 +3,15 @@
   "backButtonTooltip": "Zurück",
   "closeButtonTooltip": "Schließen ",
```
Comment (goderbauer) [inline packages/flutter/lib/src/material/i18n/material_de.arb]: Remove extra space?

File: packages/flutter/lib/src/material/i18n/material_de.arb [hunk 8]
```diff
@@ -3,5 +3,15 @@
...
   "nextMonthTooltip": "Nächster Monat",
-  "previousMonthTooltip": "Letzter Monat"
+  "previousMonthTooltip": "Letzter Monat",
```
Comment (goderbauer) [inline packages/flutter/lib/src/material/i18n/material_de.arb]: better: Vorheriger Monat

File: packages/flutter/lib/src/material/i18n/localizations.dart [hunk 9]
```diff
@@ -16,104 +16,254 @@ const Map<String, Map<String, String>> localizations = const <String, Map<String
...
+    "selectAllButtonLabel": r"SELECCIONAR TODO",
+    "viewLicensesButtonLabel": r"VER LICENCIAS"
   },
   "fr": const <String, String>{
     "openAppDrawerTooltip": r"Ouvrir le menu de navigation",
     "backButtonTooltip": r"Arrière",
     "closeButtonTooltip": r"Fermer",
     "nextMonthTooltip": r"Mois Suivant",
-    "previousMonthTooltip": r"Le mois précédent"
+    "previousMonthTooltip": r"Le mois précédent",
```
Comment (Hixie) [inline packages/flutter/lib/src/material/i18n/localizations.dart]: `Mois précédent` for consistency with nextMonthTooltip

File: packages/flutter/lib/src/material/i18n/localizations.dart [hunk 10]
```diff
@@ -16,104 +16,254 @@ const Map<String, Map<String, String>> localizations = const <String, Map<String
...
+    "licensesPageTitle": r"Licences",
+    "cancelButtonLabel": r"ANNULER",
+    "closeButtonLabel": r"FERMER",
+    "continueButtonLabel": r"CONTINUER",
+    "copyButtonLabel": r"COPIER",
+    "cutButtonLabel": r"COUPER",
+    "okButtonLabel": r"D'accord",
```
Comment (Hixie) [inline packages/flutter/lib/src/material/i18n/localizations.dart]: `OK`

**Comment by Hixie** [comment]
LGTM, though please make sure Travis runs over it first...

File: packages/flutter/lib/src/material/i18n/localizations.dart [hunk 11]
```diff
@@ -16,104 +16,254 @@ const Map<String, Map<String, String>> localizations = const <String, Map<String
...
+    "licensesPageTitle": r"Licences",
+    "cancelButtonLabel": r"ANNULER",
+    "closeButtonLabel": r"FERMER",
+    "continueButtonLabel": r"CONTINUER",
+    "copyButtonLabel": r"COPIER",
+    "cutButtonLabel": r"COUPER",
+    "okButtonLabel": r"OK",
+    "pasteButtonLabel": r"COLLER",
+    "selectAllButtonLabel": r"TOUT SÉLECTIONNER",
+    "viewLicensesButtonLabel": r"VOIR LES LICENCES"
```
Comment (cbracken) [inline packages/flutter/lib/src/material/i18n/localizations.dart]: I'd use: AFFICHER LES LICENCES

File: packages/flutter/lib/src/material/i18n/localizations.dart [hunk 12]
```diff
@@ -16,104 +16,254 @@ const Map<String, Map<String, String>> localizations = const <String, Map<String
...
   },
   "zh": const <String, String>{
     "openAppDrawerTooltip": r"打开导航菜单",
     "backButtonTooltip": r"背部",
```
Comment (xster) [inline packages/flutter/lib/src/material/i18n/localizations.dart]: probably '返回'. The current one means back as in dorsal

File: packages/flutter/lib/src/material/i18n/localizations.dart [hunk 13]
```diff
@@ -16,104 +16,254 @@ const Map<String, Map<String, String>> localizations = const <String, Map<String
...
   "ja": const <String, String>{
     "openAppDrawerTooltip": r"ナビゲーションメニューを開く",
     "backButtonTooltip": r"バック",
     "closeButtonTooltip": r"閉じる",
     "nextMonthTooltip": r"来月",
-    "previousMonthTooltip": r"前の月"
+    "previousMonthTooltip": r"前の月",
+    "licensesPageTitle": r"ライセンス",
+    "cancelButtonLabel": r"キャンセル",
+    "closeButtonLabel": r"閉",
```
Comment (cbracken) [inline packages/flutter/lib/src/material/i18n/localizations.dart]: 閉じる

File: packages/flutter/lib/src/material/i18n/localizations.dart [hunk 14]
```diff
@@ -16,104 +16,254 @@ const Map<String, Map<String, String>> localizations = const <String, Map<String
...
+    "cutButtonLabel": r"COUPER",
+    "okButtonLabel": r"OK",
+    "pasteButtonLabel": r"COLLER",
+    "selectAllButtonLabel": r"TOUT SÉLECTIONNER",
+    "viewLicensesButtonLabel": r"VOIR LES LICENCES"
   },
   "zh": const <String, String>{
     "openAppDrawerTooltip": r"打开导航菜单",
     "backButtonTooltip": r"背部",
     "closeButtonTooltip": r"关",
```
Comment (xster) [inline packages/flutter/lib/src/material/i18n/localizations.dart]: 关闭

File: packages/flutter/lib/src/material/i18n/localizations.dart [hunk 15]
```diff
@@ -16,104 +16,254 @@ const Map<String, Map<String, String>> localizations = const <String, Map<String
...
     "nextMonthTooltip": r"-下月就29了。",
```
Comment (xster) [inline packages/flutter/lib/src/material/i18n/localizations.dart]: 下一个月

File: packages/flutter/lib/src/material/i18n/localizations.dart (hunk 13 above)
Comment (HansMuller) [reply packages/flutter/lib/src/material/i18n/localizations.dart]: ありがとう！

File: packages/flutter/lib/src/material/i18n/localizations.dart [hunk 16]
```diff
@@ -16,104 +16,254 @@ const Map<String, Map<String, String>> localizations = const <String, Map<String
...
-    "previousMonthTooltip": r"前一个月"
+    "previousMonthTooltip": r"前一个月",
+    "licensesPageTitle": r"许可证",
+    "cancelButtonLabel": r"取消",
+    "continueButtonLabel": r"继续",
+    "closeButtonLabel": r"关闭",
+    "copyButtonLabel": r"副本",
+    "cutButtonLabel": r"切除",
```
Comment (xster) [inline packages/flutter/lib/src/material/i18n/localizations.dart]: 剪切

File: packages/flutter/lib/src/material/i18n/localizations.dart [hunk 17]
```diff
@@ -16,104 +16,254 @@ const Map<String, Map<String, String>> localizations = const <String, Map<String
...
+    "okButtonLabel": r"好",
+    "pasteButtonLabel": r"贴上",
```
Comment (xster) [inline packages/flutter/lib/src/material/i18n/localizations.dart]: 粘贴

File: packages/flutter/lib/src/material/i18n/localizations.dart [hunk 18]
```diff
@@ -16,104 +16,254 @@ const Map<String, Map<String, String>> localizations = const <String, Map<String
...
     "backButtonTooltip": r"背部",
     "closeButtonTooltip": r"关",
     "nextMonthTooltip": r"-下月就29了。",
-    "previousMonthTooltip": r"前一个月"
+    "previousMonthTooltip": r"前一个月",
+    "licensesPageTitle": r"许可证",
+    "cancelButtonLabel": r"取消",
+    "continueButtonLabel": r"继续",
+    "closeButtonLabel": r"关闭",
+    "copyButtonLabel": r"副本",
```
Comment (xster) [inline packages/flutter/lib/src/material/i18n/localizations.dart]: 复制

File: packages/flutter/lib/src/material/i18n/localizations.dart [hunk 19]
```diff
@@ -16,104 +16,254 @@ const Map<String, Map<String, String>> localizations = const <String, Map<String
...
+    "cutButtonLabel": r"切除",
+    "okButtonLabel": r"好",
+    "pasteButtonLabel": r"贴上",
+    "selectAllButtonLabel": r"选中所有",
```
Comment (xster) [inline packages/flutter/lib/src/material/i18n/localizations.dart]: 全选

File: packages/flutter/lib/src/material/i18n/localizations.dart [hunk 20]
```diff
@@ -16,104 +16,254 @@ const Map<String, Map<String, String>> localizations = const <String, Map<String
...
     "closeButtonTooltip": r"閉じる",
     "nextMonthTooltip": r"来月",
-    "previousMonthTooltip": r"前の月"
+    "previousMonthTooltip": r"前の月",
+    "licensesPageTitle": r"ライセンス",
+    "cancelButtonLabel": r"キャンセル",
+    "closeButtonLabel": r"閉",
+    "continueButtonLabel": r"持続する",
+    "copyButtonLabel": r"コピー",
+    "cutButtonLabel": r"切り",
```
Comment (cbracken) [inline packages/flutter/lib/src/material/i18n/localizations.dart]: カット

File: packages/flutter/lib/src/material/i18n/material_zh.arb [hunk 21]
```diff
@@ -3,5 +3,15 @@
   "backButtonTooltip": "背部",
   "closeButtonTooltip": "关",
   "nextMonthTooltip": "-下月就29了。",
```
Comment (xster) [inline packages/flutter/lib/src/material/i18n/material_zh.arb]: ditto here

File: packages/flutter/lib/src/material/i18n/localizations.dart [hunk 22]
```diff
@@ -16,104 +16,254 @@ const Map<String, Map<String, String>> localizations = const <String, Map<String
...
+    "okButtonLabel": r"OK",
+    "pasteButtonLabel": r"貼付け",
```
Comment (cbracken) [inline packages/flutter/lib/src/material/i18n/localizations.dart]: Trickier...
On Android: 貼り付け
On iOS: ペースト

File: packages/flutter/lib/src/material/i18n/localizations.dart [hunk 23]
```diff
@@ -16,104 +16,254 @@ const Map<String, Map<String, String>> localizations = const <String, Map<String
...
+    "selectAllButtonLabel": r"すべて選択",
```
Comment (cbracken) [inline packages/flutter/lib/src/material/i18n/localizations.dart]: 全選択

File: packages/flutter/lib/src/material/i18n/localizations.dart [hunk 24]
```diff
@@ -16,104 +16,254 @@ const Map<String, Map<String, String>> localizations = const <String, Map<String
...
     "openAppDrawerTooltip": r"ナビゲーションメニューを開く",
     "backButtonTooltip": r"バック",
     "closeButtonTooltip": r"閉じる",
     "nextMonthTooltip": r"来月",
-    "previousMonthTooltip": r"前の月"
+    "previousMonthTooltip": r"前の月",
+    "licensesPageTitle": r"ライセンス",
+    "cancelButtonLabel": r"キャンセル",
+    "closeButtonLabel": r"閉",
+    "continueButtonLabel": r"持続する",
```
Comment (cbracken) [inline packages/flutter/lib/src/material/i18n/localizations.dart]: A "next" button is usually: 次へ

Comment (HansMuller) [reply packages/flutter/lib/src/material/i18n/localizations.dart]: Do you meant that continueButtonLabel should be 次へ?

File: packages/flutter/lib/src/material/i18n/localizations.dart (hunk 22 above)
Comment (HansMuller) [reply packages/flutter/lib/src/material/i18n/localizations.dart]: Unless you have a preference, I will leave the existing translation (which matches Android) in place.

File: packages/flutter/lib/src/material/i18n/localizations.dart [hunk 25]
```diff
@@ -16,104 +16,254 @@ const Map<String, Map<String, String>> localizations = const <String, Map<String
...
+    "selectAllButtonLabel": r"TOUT SÉLECTIONNER",
+    "viewLicensesButtonLabel": r"VOIR LES LICENCES"
   },
   "zh": const <String, String>{
     "openAppDrawerTooltip": r"打开导航菜单",
     "backButtonTooltip": r"背部",
     "closeButtonTooltip": r"关",
     "nextMonthTooltip": r"-下月就29了。",
-    "previousMonthTooltip": r"前一个月"
+    "previousMonthTooltip": r"前一个月",
```
Comment (xster) [inline packages/flutter/lib/src/material/i18n/localizations.dart]: for symmetry 上一个月

Comment (HansMuller) [reply packages/flutter/lib/src/material/i18n/localizations.dart]: 谢谢！

File: packages/flutter/lib/src/material/i18n/localizations.dart [hunk 26]
```diff
@@ -16,104 +16,254 @@ const Map<String, Map<String, String>> localizations = const <String, Map<String
...
+    "selectAllButtonLabel": r"ALLES AUSWÄHLEN",
+    "viewLicensesButtonLabel": r"LIZENZEN ANZEIGEN"
   },
   "ja": const <String, String>{
     "openAppDrawerTooltip": r"ナビゲーションメニューを開く",
     "backButtonTooltip": r"バック",
     "closeButtonTooltip": r"閉じる",
     "nextMonthTooltip": r"来月",
-    "previousMonthTooltip": r"前の月"
+    "previousMonthTooltip": r"前の月",
```
Comment (cbracken) [inline packages/flutter/lib/src/material/i18n/localizations.dart]: If the context is something like "go to the previous/next month" then maybe:
nextMonthTooltip: 次の月へ
previousMonthTooltip: 前の月へ

File: packages/flutter/lib/src/material/i18n/localizations.dart [hunk 27]
```diff
@@ -16,104 +16,254 @@ const Map<String, Map<String, String>> localizations = const <String, Map<String
...
+    "copyButtonLabel": r"KOPIEREN",
+    "cutButtonLabel": r"AUSSCHNEIDEN",
+    "okButtonLabel": r"OK",
+    "pasteButtonLabel": r"EINFÜGEN",
+    "selectAllButtonLabel": r"ALLES AUSWÄHLEN",
+    "viewLicensesButtonLabel": r"LIZENZEN ANZEIGEN"
   },
   "ja": const <String, String>{
     "openAppDrawerTooltip": r"ナビゲーションメニューを開く",
     "backButtonTooltip": r"バック",
```
Comment (cbracken) [inline packages/flutter/lib/src/material/i18n/localizations.dart]: 戻る

File: packages/flutter/lib/src/material/i18n/localizations.dart [hunk 28]
```diff
@@ -16,104 +16,254 @@ const Map<String, Map<String, String>> localizations = const <String, Map<String
...
+    "licensesPageTitle": r"ライセンス",
+    "cancelButtonLabel": r"キャンセル",
+    "closeButtonLabel": r"閉",
+    "continueButtonLabel": r"持続する",
+    "copyButtonLabel": r"コピー",
+    "cutButtonLabel": r"切り",
+    "okButtonLabel": r"OK",
+    "pasteButtonLabel": r"貼付け",
+    "selectAllButtonLabel": r"すべて選択",
+    "viewLicensesButtonLabel": r"ライセンスの表示"
```
Comment (cbracken) [inline packages/flutter/lib/src/material/i18n/localizations.dart]: ライセンス表記

File: packages/flutter/lib/src/material/i18n/localizations.dart (hunk 25 above)
Comment (HansMuller) [reply packages/flutter/lib/src/material/i18n/localizations.dart]: Looks like Google translate was 6/15 for simplified chinese.

File: packages/flutter/lib/src/material/i18n/localizations.dart (hunk 27 above)
Comment (HansMuller) [reply packages/flutter/lib/src/material/i18n/localizations.dart]: It's the BackButton's tooltip so I'll use  戻る

File: packages/flutter/lib/src/material/i18n/localizations.dart (hunk 28 above)
Comment (HansMuller) [reply packages/flutter/lib/src/material/i18n/localizations.dart]: It looks like Google translate was 9/15 for japanese.

File: packages/flutter/lib/src/material/i18n/localizations.dart (hunk 22 above)
Comment (cbracken) [reply packages/flutter/lib/src/material/i18n/localizations.dart]: The existing one *almost* matches the Android one. You're missing a り.

File: packages/flutter/lib/src/material/i18n/material_ja.arb [hunk 29]
```diff
@@ -1,7 +1,17 @@
...
+  "backButtonTooltip": "戻る",
   "closeButtonTooltip": "閉じる",
-  "nextMonthTooltip": "来月",
-  "previousMonthTooltip": "前の月"
+  "nextMonthTooltip": "前の月へ",
+  "previousMonthTooltip": "前の月へ",
+  "licensesPageTitle": "ライセンス",
+  "cancelButtonLabel": "キャンセル",
+  "closeButtonLabel": "閉じる",
+  "continueButtonLabel": "持続する",
```
Comment (cbracken) [inline packages/flutter/lib/src/material/i18n/material_ja.arb]: 次へ

File: packages/flutter/lib/src/material/i18n/material_fr.arb [hunk 30]
```diff
@@ -3,5 +3,15 @@
   "backButtonTooltip": "Arrière",
   "closeButtonTooltip": "Fermer",
   "nextMonthTooltip": "Mois Suivant",
```
Comment (cbracken) [inline packages/flutter/lib/src/material/i18n/material_fr.arb]: Should this be "Mois suivant" for consistent capitalisation with 'Mois précédent' below?

File: packages/flutter/lib/src/material/i18n/localizations.dart [hunk 31]
```diff
@@ -16,104 +16,254 @@ const Map<String, Map<String, String>> localizations = const <String, Map<String
...
+    "closeButtonTooltip": r"关闭",
+    "nextMonthTooltip": r"下一个月",
+    "previousMonthTooltip": r"上一个月",
+    "licensesPageTitle": r"许可证",
+    "cancelButtonLabel": r"取消",
+    "continueButtonLabel": r"继续",
+    "closeButtonLabel": r"关闭",
+    "copyButtonLabel": r"复制",
+    "cutButtonLabel": r"剪切",
+    "okButtonLabel": r"好",
```
Comment (xster) [inline packages/flutter/lib/src/material/i18n/localizations.dart]: On second thought I think this is too colloquial. The standard tone should be 确定

File: packages/flutter/lib/src/material/i18n/material_fr.arb [hunk 32]
```diff
@@ -3,5 +3,15 @@
   "backButtonTooltip": "Arrière",
```
Comment (cbracken) [inline packages/flutter/lib/src/material/i18n/material_fr.arb]: Retour

e.g., normally "bouton retour" for the "back button"

File: packages/flutter/lib/src/material/i18n/localizations.dart [hunk 33]
```diff
@@ -16,104 +16,254 @@ const Map<String, Map<String, String>> localizations = const <String, Map<String
...
+    "selectAllButtonLabel": r"SELECIONAR TUDO",
+    "viewLicensesButtonLabel": r"VER LICENÇAS"
   },
   "es": const <String, String>{
     "openAppDrawerTooltip": r"Abrir el menú de navegación",
     "backButtonTooltip": r"Espalda",
     "closeButtonTooltip": r"Cerrar",
     "nextMonthTooltip": r"Próximo mes",
-    "previousMonthTooltip": r"mes anterior"
+    "previousMonthTooltip": r"mes anterior",
```
Comment (cbracken) [inline packages/flutter/lib/src/material/i18n/localizations.dart]: Capital 'Mes' for consistency with the line below?

File: packages/flutter/lib/src/material/i18n/localizations.dart [hunk 34]
```diff
@@ -16,104 +16,254 @@ const Map<String, Map<String, String>> localizations = const <String, Map<String
...
+    "copyButtonLabel": r"コピー",
+    "cutButtonLabel": r"カット",
+    "okButtonLabel": r"OK",
+    "pasteButtonLabel": r"貼付け",
+    "selectAllButtonLabel": r"全選択",
+    "viewLicensesButtonLabel": r"ライセンス表記"
   },
   "ru": const <String, String>{
     "openAppDrawerTooltip": r"Открыть меню навигации",
     "backButtonTooltip": r"назад",
```
Comment (aam) [inline packages/flutter/lib/src/material/i18n/localizations.dart]: "Назад" - first letter should be uppercased

File: packages/flutter/lib/src/material/i18n/localizations.dart [hunk 35]
```diff
@@ -16,104 +16,254 @@ const Map<String, Map<String, String>> localizations = const <String, Map<String
...
     "closeButtonTooltip": r"Закрыть",
     "nextMonthTooltip": r"В следующем месяце",
-    "previousMonthTooltip": r"Предыдущий месяц"
+    "previousMonthTooltip": r"Предыдущий месяц",
+    "licensesPageTitle": r"Лицензии",
+    "cancelButtonLabel": r"ОТМЕНА",
```
Comment (aam) [inline packages/flutter/lib/src/material/i18n/localizations.dart]: "Отменить" seems to be more appropriate here. It is a verb, while "Отмена" is a noun(=cancellation).

File: packages/flutter/lib/src/material/i18n/localizations.dart [hunk 36]
```diff
@@ -16,104 +16,254 @@ const Map<String, Map<String, String>> localizations = const <String, Map<String
...
+    "closeButtonLabel": r"ЗАКРЫТЬ",
```
Comment (aam) [inline packages/flutter/lib/src/material/i18n/localizations.dart]: Is all upper-case intentional here(and below)?

File: packages/flutter/lib/src/material/i18n/localizations.dart [hunk 37]
```diff
@@ -16,104 +16,254 @@ const Map<String, Map<String, String>> localizations = const <String, Map<String
...
+    "continueButtonLabel": r"ПРОДОЛЖАТЬ",
```
Comment (aam) [inline packages/flutter/lib/src/material/i18n/localizations.dart]: "Продолжить"

File: packages/flutter/lib/src/material/i18n/localizations.dart [hunk 38]
```diff
@@ -16,104 +16,254 @@ const Map<String, Map<String, String>> localizations = const <String, Map<String
...
+    "copyButtonLabel": r"копия",
```
Comment (aam) [inline packages/flutter/lib/src/material/i18n/localizations.dart]: "Скопировать"

File: packages/flutter/lib/src/material/i18n/localizations.dart [hunk 39]
```diff
@@ -16,104 +16,254 @@ const Map<String, Map<String, String>> localizations = const <String, Map<String
...
+    "cutButtonLabel": r"порез",
```
Comment (aam) [inline packages/flutter/lib/src/material/i18n/localizations.dart]: "Вырезать"

File: packages/flutter/lib/src/material/i18n/localizations.dart [hunk 40]
```diff
@@ -16,104 +16,254 @@ const Map<String, Map<String, String>> localizations = const <String, Map<String
...
+    "okButtonLabel": r"ОК",
+    "pasteButtonLabel": r"Паст",
```
Comment (aam) [inline packages/flutter/lib/src/material/i18n/localizations.dart]: "Вставить"

File: packages/flutter/lib/src/material/i18n/localizations.dart [hunk 41]
```diff
@@ -16,104 +16,254 @@ const Map<String, Map<String, String>> localizations = const <String, Map<String
...
+    "selectAllButtonLabel": r"Выбрать все",
+    "viewLicensesButtonLabel": r"ПРОСМОТРЕТЬ ЛИЦЕНЗИИ"
```
Comment (aam) [inline packages/flutter/lib/src/material/i18n/localizations.dart]: All upper case looks funny(inconsistent?)

Comment (Hixie) [reply packages/flutter/lib/src/material/i18n/localizations.dart]: In theory all button labels should be uppercase, I guess the error is with the others.

**Pull request merged**